src/cljs/ataru/application_common/application_field_common.cljs

```diff
@@ -77,6 +80,11 @@
 
       :else value-or-koodi-uri)))
 
+(defn group-spacer
+  [index]
+  [^{:key (str "spacer-" index)}
```

Comment:
Isn't the outer vector in the return value here unnecessary? Just returning

```clojure
^{:key (str "spacer-" index)}
[:div.application__question-group-spacer]
```

should be enough?

Reply:
Yeah, fixed by the next comment!

Reply:
Yeah!